pyodide_build/buildpkg.py

```diff
@@ -103,7 +104,7 @@ def download_and_extract(buildpath, packagedir, pkg, args):
         raise ValueError("Incorrect source provided")
 
 
-def patch(path, srcpath, pkg, args):
+def patch(path: Path, srcpath: Path, pkg: Dict[str, Any], args: List(Any)):
```

Comment:
I think `List(Any)` is syntactically not correct. It should be `List[Any]` or just `List` actually.